### DISCUSSION TITLE
Track cost of charging session

### DISCUSSION THREAD

**Discussion by TonyTigern** [lbbrhzn/ocpp#876] [Ideas]
Hi.

What would be the best way to track the cost for a charging session? Right now i have a sensor tracking the current energy cost with the Nordpool integration, where the state contains the current cost, and has attributes for past and future costs by the hour. Would it be much work to incorporate a sensor with this integration to track the cost for the current charging session given a cost sensor, and that resets whenever a new session starts?

**Comment by drc38** [comment]
https://community.home-assistant.io/t/solved-calculate-cost-for-individual-device-cost-kwh/523880/3
